ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -65,10 +65,14 @@ literals:
   - OFF
 - discharge_fan_speed_mode:
   - AUTO
+  - AUTOLOW
+  - AUTOMEDIUM
+  - AUTOHIGH
   - LOW
   - MEDIUM
   - HIGH
   - OFF
+  - VFD
```

Comment:
Why is this here?

Reply:
When a user switches to this mode VFD, the user can adjust the fan_speed_frequency_command or fan_speed_percentage_command.  We had this discussion before in Google meet, the FCU FCU_DFVDSFC_CHWBRC was the one with this function.